### PR TITLE
New FV3_GFS_v16_gsd_chem CCPP suite

### PR DESCRIPTION
**Pull request by SamuelTrahanNOAA**
The gsd-chem development is switching to the new gfs v16 suite. This adds a gsd-chem coupled version of that suite.

Corresponding ufs-weather-model PR:

https://github.com/NOAA-GSL/ufs-weather-model/pull/78

### CODE REVIEW

**Review by mpagowski — APPROVED** [review]

**Review by mpagowski — APPROVED** [review]

**Comment by climbfuji** [comment]
@SamuelTrahanNOAA @mpagowski @zhanglikate @haiqinli I don't want to interfere with your code management practices, but given that @zhanglikate and @haiqinli are working off of the same branch on the NASA GOCART integration, I would suggest to request and wait for approval from at least one person from each development stream for anything that goes into gsl/develop-chem.

**Comment by SamuelTrahanNOAA** [comment]
Haiqin and Li asked me to add these updates last week.

**Comment by DomHeinzeller** [comment]
Thanks, that was not clear from the PR. You can always add them as reviewers, they should be available from the pulldown menu.

**Comment by haiqinli** [comment]
I would like to approve this PR. Thanks.

**Comment by SamuelTrahanNOAA** [comment]
By the way, you need to set these two variables to get the right input data and baseline. I'm not sure how to correctly set this in the rt.sh since the paths are calculated in three steps. Ideally, these should be moved to a common area rather than my account.

  export RTPWD=/scratch2/BMC/wrfruc/Samuel.Trahan/perturb-smoke/RT/NEMSfv3gfs/develop-20210330/INTEL
  export INPUTDATA_ROOT=/scratch2/BMC/wrfruc/Samuel.Trahan/perturb-smoke/RT/NEMSfv3gfs/input-data-20210324

**Comment by DomHeinzeller** [comment]
None of this can survive when merging into the authoritative code repository. We'll need to copy the input data into the official locations and get the baselines updated.

As I said last week, the easiest is to not maintain a permanent baseline but simply generate a temporary one using the head of gsl/develop-chem with the "-c" flag for rt.sh, and then verify the new code against it with the "-m" flag.

I guess you only have a handful of input data files, probably easiest to explicitly copy those in the regression test run scripts themselves and not modify INPUTDATA_ROOT.

**Comment by SamuelTrahanNOAA** [comment]
I created the baselines as you describe. My INPUTDATA_ROOT is a copy of the entire input data root from the upstream, but with the chemistry inputs added in two subdirectories. Likewise for the baseline, with four new subdirectories.

**Pull request merged**
